src/assets/posts.json

```diff
@@ -1,25 +1,23 @@
-{
-    "posts": [
-        {
-            "id": 1,
-            "title": "První post",
-            "date_created": "2014-01-01T23:28:56.782Z",
-            "subreddit": "Subreddit1",
-            "author": "Pepin",
-            "commentCount": 2,
-            "post_type": "text",
-            "description": "Post o ničem"
-        },
-        {
-            "id": 2,
-            "title": "Druhý post",
-            "date_created": "2020-11-30T17:10:56.782Z",
-            "subreddit": "Subreddit2",
-            "author": "Pepinovič",
-            "commentCount": 3,
-            "post_type": "url",
-            "posted_url": "https://seznam.cz",
-            "description": "Post úplně o ničem"
-        }
-    ]
-}
\ No newline at end of file
+[
+    {
+        "id": 1,
+        "title": "First post",
+        "date_created": "2014-01-01T23:28:56.782Z",
+        "subreddit": "Subreddit1",
+        "author": "Author1",
+        "commentCount": 2,
+        "post_type": "text",
+        "description": "Post about something"
+    },
+    {
+        "id": 2,
+        "title": "Second post",
+        "date_created": "2020-11-30T17:10:56.782Z",
+        "subreddit": "Subreddit2",
+        "author": "Author2",
+        "commentCount": 3,
+        "post_type": "url",
+        "posted_url": "https://seznam.cz",
+        "description": "Post about anything"
+    }
+]
```

Comment:
Do you need this? Since you are mocking posts in tests and feed is functional on backend.

Reply:
If it wouldn't mind you I would keep it since it helps me to develop frontend faster (because of my slow computer).

Reply:
You should not use it like this, though. Instead, you should use only one. It makes sense in some big data cases that you are using mocked data for development rather than real data as well. 

Think about using mocked data on your machine only and doing it sustainably so you don't need to change that every time.

Reply:
I deleted both files. I haven't figured out how to use mocked data only on my machine, but it works with with backend acceptably.